src/App.tsx

```diff
@@ -63,10 +62,10 @@ export const App: React.FC = () => {
       userId: selectedUserId,
     };
 
-    if (trimTitle && selectedUserId) {
-      setTodos((currentTodos) => [...currentTodos, newTodo]);
-      resetForm();
-    }
+    todosWithUsers.push(newTodo);
```

Comment:
This needs to be is state and have corresponding setter function. Or use your `todos` here. But set it as state value.